.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
-  "useTabs": true,
-  "tabWidth": 4,
+  "useTabs": false,
+  "tabWidth": 2,
```

Comment:
Not sure with these !!
do you have any strong reason for these changes?

Reply:
cc @elsonlim

Reply:
The test pages are using 2 spaces indentation, I realise I change that because 2 spaces indentation is used in all test files, now looking at other files, I realise we didn't standardise. For example, appRouter is using 2 spaces, app switch is using 4 spaces, some jsx files are using tabs. I will remove this change. Moving forward I guess we should standardise it in all pages and have eslint runs in the PR checks

Reply:
good catch 👍 
Let's make all as 4 indent space! 👍 
or if you have strong reason for 2, then we can go with that too.